src/map/glmap/gl-styles/style.json

```diff
@@ -546,104 +573,161 @@
+        "circle-radius": 3,
+        "circle-color": "#FFE7A0",
+        "circle-stroke-width": 0.5,
+        "circle-stroke-color": "#000000"
       }
     },
     {
-      "id": "events_gap",
+      "id": "events_port",
       "type": "circle",
-      "source": "events_gap",
+      "source": "events_port",
       "source-layer": "default",
       "metadata": {
         "gfw:temporal": true,
         "mapbox:group": "temporal"
       },
       "paint": {
         "circle-radius": 3,
-        "circle-color": "#FFE7A0",
+        "circle-color": "#44A1B9",
         "circle-stroke-width": 0.5,
         "circle-stroke-color": "#000000"
       }
     },
     {
-      "id": "events_gap_icons",
+      "id": "events_encounter_vessel",
+      "source": "events_encounter_vessel",
       "type": "symbol",
       "metadata": {
-        "gfw:temporal": true,
+        "mapbox:group": "temporal",
         "gfw:styles": {
-          "selected": {
-            "icon-size": [
-              1,
-              0
+          "highlighted": {
+            "icon-image": [
+              "encounter_highlight",
+              "encounter"
             ]
           }
-        },
+        }
+      },
+      "layout": {
+        "icon-image": "encounter",
+        "icon-allow-overlap": true
+      }
+    },
+    {
+      "id": "events_encounter_vessel_number",
+      "source": "events_encounter_vessel",
+      "type": "symbol",
+      "metadata": {
         "mapbox:group": "temporal"
       },
-      "source": "events_gap",
-      "source-layer": "default",
+      "filter": ["has", "point_count"],
       "layout": {
-          "icon-image": "gap",
-          "icon-size": 0,
-          "icon-allow-overlap": true
+        "text-field": "{point_count_abbreviated}",
+        "text-font": [
+          "Roboto Medium"
+        ],
+        "text-size": 12,
+        "text-anchor": "bottom-left",
+        "text-offset": [0.8, -0.2]
+      },
+      "paint": {
+        "text-halo-color": "hsl(0, 0%, 100%)",
+        "text-halo-width": 2
       }
     },
     {
-      "id": "events_port",
-      "type": "circle",
-      "source": "events_port",
-      "source-layer": "default",
+      "id": "events_gap_vessel",
+      "source": "events_gap_vessel",
+      "type": "symbol",
+      "metadata": {
+        "mapbox:group": "temporal",
+        "gfw:styles": {
+          "highlighted": {
+            "icon-image": [
+              "gap_highlight",
+              "gap"
+            ]
+          }
+        }
+      },
+      "layout": {
+        "icon-image": "gap",
+        "icon-allow-overlap": true
+      }
+    },
+    {
+      "id": "events_gap_vessel_number",
+      "source": "events_gap_vessel",
+      "type": "symbol",
       "metadata": {
-        "gfw:temporal": true,
         "mapbox:group": "temporal"
       },
+      "filter": ["has", "point_count"],
+      "layout": {
+        "text-field": "{point_count_abbreviated}",
+        "text-font": [
+          "Roboto Medium"
```

Comment:
Is this working? I mean is the font included in the map component or depends on the project where the component is used ?

Reply:
Font families are those available after generation of the glyphs. See https://github.com/GlobalFishingWatch/map-gl-glyphs

Reply:
Thanks!